proto/kava/auction/v1beta2/genesis.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package kava.auction.v1beta2;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/kava-labs/kava/x/auction/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// GenesisState defines the auction module's genesis state.
+message GenesisState {
+  uint64 next_auction_id = 1;
+
+  Params params = 2 [(gogoproto.nullable) = false];
+
+  // Genesis auctions
+  repeated google.protobuf.Any auctions = 3 [(cosmos_proto.accepts_interface) = "GenesisAuction"];
+}
+
+// Params defines the parameters for the issuance module.
+message Params {
+
+  google.protobuf.Duration max_auction_duration = 1 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+
+  google.protobuf.Duration forward_bid_duration = 6 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
```

Comment:
@nddeluca @drklee3 since the previous change hasn't made it out to prod yet, decided to fix the auction params proto update to be backwards compatible (e.g. not re-using the previous field value of 2 for forward_bid_duration)

https://developers.google.com/protocol-buffers/docs/proto3#updating

https://github.com/Kava-Labs/kava/pull/1158/files#diff-7d962fdfc749cc7bd20b415e063b23cc20cf6458af6d18b09445fef7654ee323L28